docs/pages/getting-started/authentication/credentials.mdx

```diff
@@ -217,6 +217,49 @@ export function SignIn() {
 ```
 
 </Code.Next>
+<Code.NextClient>
+
+```tsx filename="./components/sign-in.tsx"
+"use client"
+import { useState, ChangeEvent, FormEvent } from "react"
+import { signIn } from "next-auth/react"
+
+export function SignIn() {
+  const [credentials, setCredentials] = useState({
+    email: "",
+    password: "",
+  })
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target
+    setCredentials({
+      ...credentials,
+      [name]: value
+    })
+  }
```

Comment:
Let's drop these and follow the latest recommendation from the React docs: https://react.dev/reference/react-dom/components/form#handle-form-submission-on-the-client.

Reply:
I have made the changes based on your suggestions and updated the code to work with the latest React version, particularly for handling forms. I will wait for your response.